app/models/manageiq/providers/automation_manager/orchestration_stack.rb

```diff
@@ -1,2 +1,7 @@
 class ManageIQ::Providers::AutomationManager::OrchestrationStack < ::OrchestrationStack
+  include CiFeatureMixin
+
+  def retireable?
+    false
+  end
```

Comment:
Looks good.  

Let's add tests to validate that the subclassed `Job` instances properly reflect this method override.
`ManageIQ::Providers::AnsibleTower::AutomationManager::Job`
`ManageIQ::Providers::AnsibleTower::AutomationManager::WorkflowJob`
`ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job`